eng/scripts/vs.buildtools.json

```diff
@@ -25,6 +25,7 @@
         "Microsoft.VisualStudio.Workload.NetCoreBuildTools",
         "Microsoft.VisualStudio.Workload.VCTools",
         "Microsoft.VisualStudio.Workload.VisualStudioExtensionBuildTools",
-        "Microsoft.VisualStudio.Workload.WebBuildTools"
+        "Microsoft.VisualStudio.Workload.WebBuildTools",
+        "Microsoft.VisualStudio.Component.IISExpress"
```

Comment:
Why is this still changing? I would have thought #9120 was the way to go since this change didn't work.